wp-content/themes/core/assets/css/src/theme/media-queries/_variables.pcss

```diff
@@ -10,6 +10,7 @@
 @custom-media --viewport-xxsmall (min-width: 320px);
 @custom-media --viewport-xsmall (min-width: 500px);
 @custom-media --viewport-small (min-width: 600px);
+@custom-media --viewport-medium-max (max-width: 767px);
```

Comment:
Is this worth creating a variable for what is currently a single use case?

Reply:
I'm fine with this, but I think it should be named `--viewport-small-max`. Imho, `--viewport-medium-max` would be 959px. :)